.github/workflows/datadog-trace-verify.yml

```
@@ -0,0 +1,47 @@
+          fetch-depth: 1
+
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install dependencies
+        env:
+          NPM_CONFIG_IGNORE_SCRIPTS: "true"
+          npm_config_ignore_scripts: "true"
```

actionlint (1.7.7)

key "npm_config_ignore_scripts" is duplicated in env. previously defined at line:26,col:11. note that this key is case insensitive

(syntax-check)